### PR TITLE
Optimized the options and implemented the reserved_case option

### PR DESCRIPTION
**Pull request by Shun7**
Optimized the structure of the options and implemented the reserved_case option to be able to change case of reserved words.

### CODE REVIEW

**Comment by Shun7** [comment]
Tested on my local pc and found some bugs. Now it works fine. So please check the code and pull the code!

**Comment by Shun7** [comment]
@ota-meshi 
Added the case, reserved case and reserved word options instead of just the uppercase option. Also updated the readme files. Could you please check the code and pull the request if everything is ok.

File: urobosql_formatter.py [hunk 1]
```diff
@@ -24,13 +24,25 @@ def run(self, edit, sync=False):
...
         self.settings.set("tab_size", self.getval("uf_tab_size"))
         self.settings.set("translate_tabs_to_spaces",
                           self.getval("uf_translate_tabs_to_spaces"))
-
         config = LocalConfig()
-        config.set_uppercase(self.getval("uf_uppercase"))
+        if self.getval("uf_case") == 'nochange':
+            config.set_case(None)
+        else:
+            config.set_case(self.getval("uf_case"))
```
Comment (ota-meshi) [inline urobosql_formatter.py]: For backward compatibility if 
```py
self.user_settings.get("uf_uppercase") == False and self.user_settings.get("uf_upper") == None
```
, please behavior as 
```py
config.set_case(None)
```

Comment (ota-meshi) [reply urobosql_formatter.py]: The same applies to `"uf_reserved_case"`

**Comment by Shun7** [comment]
@ota-meshi 
Modified the code to manage backward compatibility. Please review the code again.

**Comment by ota-meshi** [comment]
close #4

**Pull request merged**